mathics/builtin/drawing/uniform_polyhedra.py

```diff
@@ -96,11 +111,15 @@ class Tetrahedron(Builtin):
       <dd>a regular tetrahedron centered at the origin with unit edge length.
     </dl>
 
-    >> Graphics3D[Tetrahedron[]]
+    >> Graphics3D[Tetrahedron[{{0,0,0}, {1,1,1}}, 2]]
```

Comment:
The edge length difference is only seen when you show the axes or other elements in the scene.

Reply:
Ok - example extended. It also shows a bug in how axes are shown though.